github_checkpatch.py

```diff
@@ -39,24 +39,30 @@
 import sys
 import subprocess
 import re
-import requests
 import ssl
+import requests
 from github import Github
 from github import GithubException
 
 # need to monkey-patch the dismiss method until it's in a release
 def pygithub_dismiss(self, message):
+    # pylint: disable=line-too-long
     """
     :calls: `PUT /repos/:owner/:repo/pulls/:number/reviews/:review_id/dismissals <https://developer.github.com/v3/pulls/reviews/>`_
     :rtype: None
     """
+    # pylint: enable=line-too-long
     assert isinstance(message, (str, unicode)), message
     post_parameters = {'message': message}
+    # pylint: disable=unused-variable
```

Comment:
if you add an _ before whichever variable it complains about, pylint will ignore it

Reply:
Just use _ for the variable name is the recommended solution.

Reply:
This is code we are monkey-patching in from the [pygithub project](https://github.com/PyGithub/PyGithub).  I was temped to even just remove the assignment to the unused variables but I wanted to leave the code as pristine from the upstream project as possible

I can still use _ mechanism if you still think that is worth doing.

Reply:
I had to look up what monkey-patching was.  For code-review code I don't think it makes much difference, and I hadn't realised this was effectively a fork of upstream.  In general I think we should fix pylint warnings rather than suppress them and the _ mechanism is the recommended way of fixing unused-variable warnings, but I'm happy that this particular case is distinct enough to warrant an exemption.

Reply:
It's not a fork of upstream.  It's pulling in a feature that has landed upstream that is not available in a release yet.  When a future release with this function in it becomes available, it will be removed from this source and the upstream copy will be used.